assurance/design-assurance.md

```diff
@@ -7,28 +7,77 @@ expires: 2022-07-01
 
 > Architecture in DfE has moved to a more profession-led model. This content is under review. Contact the [architecture profession](mailto:architecture.profession@education.gov.uk) for more information.
 
-## In-line assurance
+## Technical design assurance
```

Comment:
I'd envisaged 'technical design assurance' being the H1... which will impact all following headings - can we consider that please?